chains/orchestrator-paras/node/src/cli.rs

```diff
@@ -94,11 +94,11 @@ pub struct SoloChainCmd {
 
 #[derive(Debug, Clone, clap::Args)]
 pub struct BuildSpecCmdExtraFields {
-    /// List of container chain chain spec paths to add to genesis.
+    /// List of container chain spec paths to add to genesis.
```

Comment:
Basically, this is not a duplicate, Container chain chain_spec file, this is what is meant here.
So, this is correct.